newIDE/app/src/EventsSheet/index.js

```diff
@@ -315,6 +315,10 @@ export class EventsSheetComponentWithoutHandle extends React.Component<
     }
   }
 
+  onResourceExternallyChanged = resourceInfo => {
```

Comment:
resourceInfo is not used it seems, so the function could be simplified?

Reply:
It's for Flow not to raise